config.js

```diff
@@ -119,6 +119,12 @@ config.smtp = {
      pass: 'xyz'
      }
     */
+    ,
+    tls: {
```

Comment:
Should the new option be document in some .md? (not sure to which level of deatil we are describing configuration in .mds... to be checked)

Reply:
Doc was updated.